about/index.md

```diff
@@ -6,29 +6,61 @@ title: About MineRL
 achieve Gold/Platinum MMR (~50% percentile human performance) using pretraining alone. We believe that leveraging human
 data will be an important piece of the puzzle as we tackle sample efficiency in more and more complex problems.
 
-However, despite the increasing number of benchmark RL environments, there has been a lack of large-scale imitation learning
-datasets. Thus, we release **MineRL**: a large-scale dataset on Minecraft of seven different tasks, which highlight
-a variety of research challenges including open-world multi-agent interactions, long-term planning, vision, control,
-navigation, and explicit and implicit subtask hierarchies. We also release a platform on which to collect more data:
-a Minecraft mod that records human trajectories, a public Minecraft server with minigames/Survival play with this mod,
-and a flexible framework to define new Minecraft tasks.
+The [Diamond competition](/diamond) is particularly focused on this challenge.
+
+### Difficulty of specifying tasks
+
+It is hard to specify reward functions for many realistic tasks: for example, how would you define a reward function for
+washing the dishes, when all you have access to are pixel inputs? Recent research has proposed alternative task
+specifications, such as learning from [demonstrations](https://arxiv.org/abs/1606.03476),
+[comparisons](https://openai.com/blog/deep-reinforcement-learning-from-human-preferences/), [reward
+signals](https://arxiv.org/abs/1709.10163), [advantage signals](https://arxiv.org/abs/1701.06049), etc.
+
+To properly evaluate the use of such techniques for task specification, we need environments in which tasks are hard to
+specify. This happens in open world environments, where there are many realistic goals that an agent could pursue. Current
+benchmarks do not meet this standard: for example, [you can get to a third of expert performance on the MuJoCo Hopper with a
+constant reward](http://arxiv.org/abs/1809.02925), and on Atari [purely curious agents perform quite
+well](https://pathak22.github.io/large-scale-curiosity/), despite having no access to the true reward function.
+
+The [BASALT competition](/basalt) is particularly focused on this challenge.
 
 Why Minecraft?
 --------------
 
 Minecraft is a rich environment to do RL on: it is an open-world environment, has sparse rewards, and has many innate
-task hierarchies and subgoals. Furthermore, it encompasses many of the problems that we must solve as we move towards
-more general AI (for example, what is the reward structure of "building a house"?). Besides all this, Minecraft has
-more than 90 million monthly active users, making it a good environment on which to collect a large-scale dataset.
+task hierarchies and subgoals. People can have many different goals within the game: perhaps you want to [defeat the Ender
+Dragon while others stop you](https://www.youtube.com/watch?v=tylNqtyj0gs), or [build a giant floating island
```

Comment:
Maybe should be "while others try to stop you," since if they just immediately succeed I imagine that isn't very fun :P

Reply:
Lol, sounds right, fixed